Makefile

```diff
@@ -18,7 +18,7 @@ CI_TEST_ARGS?=
 XC_PUBLISH?=
 PUBLISH_LOCATION?=https://releases.hashicorp.com
```

Comment:
Might be able to remove some of these vars if they're not being used anymore:
- GOLANG_IMAGE
- XC_PUBLISH
- PUBLISH_LOCATION

Reply:
👍 good spot, GOOS and GOARCH were actually unused too (double checked they weren't being used for the `build` target when run on non-linux).